packages/api/src/api/videos.ts

```diff
@@ -4,9 +4,17 @@ import logger from '../logger';
 
 export const videos = Router();
 
-videos.get('', async (req, res) => {
+videos.get('', async (req, res) => { 
+  const query = req.query.q || '';
+  if(typeof query !== 'string'){
+    res.status(400).send('query must be string');
+    return;
+  }
```

Comment:
Make sure there's a blank line between the end of this `if` and the `try` after it